common.blocks/i-bem/i-bem.vanilla.js

```diff
@@ -182,6 +182,8 @@ function declEntity(baseCls, entityName, base, props, staticProps) {
     return entityCls;
 }
 
+// jscs:disable requireMultipleVarDecl
```

Comment:
Why do we have to disable `requireMultipleVarDecl`. We don't use it in our lint rules, do we?